emspayafterpay/emspayafterpay.php

```diff
@@ -244,7 +265,25 @@ protected function isSetShowForIpFilter()
         }
         return false;
     }
-    
+
+    /**
+     * check if the EMS_AFTERPAY_COUNTY_ACCESS is set,
+     * if so, only display if user is from that counties
+     *
+     * @return boolean
+     */
+    protected function CountryAccess($idusercountry)
+    {
+        $ems_afterpay_country_access = Configuration::get('EMS_AFTERPAY_COUNTRY_ACCESS');
+        if (strlen($ems_afterpay_country_access)) {
+            $countrylist =  explode(",",str_replace(' ','',$ems_afterpay_country_access));
+            if (in_array($this->getUserCountryFromAddressId($idusercountry), $countrylist)) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
You could greatly simplify your code
the last lines using the value returned by the in_array function. Llike this:
return (if statment)